### DISCUSSION TITLE
Issue Label Emoji Spacing not Rendering

### DISCUSSION THREAD

**Discussion by LukeWolfGames** [community/community#23199] [New to GitHub]
<p>Hello,</p>
<p>For those that use emojis in their GitHub labels like so:</p>
<p><img src="https://user-images.githubusercontent.com/28833199/181075559-86fd0c8a-2b18-4f1f-9379-1a8faaa416a3.png" alt="image" data-base62-sha1="aXLH8OKIMr5K7o5yjmatBMwr3A4" width="251" height="144"></p>
<p>It doesn’t appear like the space of the emoji is being registered in text.</p>
<p>I don’t know exactly how to report this. I have a feeling this has something to do with Chrome updating yesterday?</p>
<p>Anyone else experiencing this problem and if not, any ideas how to fix or is this something that GitHub has to fix?</p>

**Comment by LukeWolfGames** [comment]
<p><a class="mention" href="/ciami7">@ciami7</a> I don’t find your reply at all productive or constructive. I recommend you first learn better social etiquette before making replies to people’s posts.</p>

**Comment by HerbCaudill** [comment]
<p>Emoji spacing is messed up on all rendered markdown. This rule seems to be responsible:</p>
<pre><code class="lang-css">@media (-webkit-max-device-pixel-ratio: 1)
g-emoji {
  display: inline-block;
  width: 1ch;
}
</code></pre>
<p>If I disable that rule in dev tools (or override it in a user stylesheet), emoji display correctly.</p>
<p>I can also make emoji display correctly by moving the browser to a display. I’m not familiar with the <code>webkit-max-device-pixel-ratio</code> media query, but it looks like there are better alternatives.</p>
<p>Tested in Chrome &amp; Firefox; Windows 10.</p>

**Comment by LukeWolfGames** [comment]
<p>Ahh so a problem with webkit, I see. I’ve never touched any of that so hard to know where to go for that.</p>

**Comment by LukeWolfGames** [comment, marked as answer]
<p>Github seems to have now fixed this issue. Thanks for your support anyway! Nice work GitHub!</p>

**Comment by bewuethr** [comment]
<p>I still see no blank after emoji, for what it’s worth (on Chrome), for example in PR comments:</p>
<p>editor shows</p>
<p><img src="https://user-images.githubusercontent.com/8521043/181075564-aacb3731-f30d-4b51-9e4a-81d94eb93192.png" alt="image" data-base62-sha1="aox0vS7KGDVp6hU8dlsULxQs3o0" width="175" height="94"></p>
<p>renders as</p>
<p><img src="https://user-images.githubusercontent.com/8521043/181075565-19cb6841-d283-4e29-9fef-b4fa55fd57d7.png" alt="image" data-base62-sha1="bu5X9PXxDhhzYBpba5y15amg3HO" width="181" height="62"></p>

**Comment by omBratteng** [comment]
<p>I think there’s an issue again with it in Chrome.<br>
<img src="https://user-images.githubusercontent.com/1681525/181075568-62836f1e-3108-4c01-a31e-f6ca102e0739.png" alt="image" data-base62-sha1="yyhAASh2xNvwxh2fpkqrWZ5jGyY" width="119" height="43"></p>
<p>Its size is 12x12 in Chrome,<br>
<img src="https://user-images.githubusercontent.com/1681525/181075570-8855bc7d-dbb0-40ba-9a56-406788f3ca5d.png" alt="image" data-base62-sha1="37e7hiRrC38Apv3zQCiXmr54gzW" width="313" height="195"></p>
<p>Whilst 16x12 in Safari<br>
<img src="https://user-images.githubusercontent.com/1681525/181075571-ec4f96d0-83a7-49cc-b052-a8e9594c3875.png" alt="image" data-base62-sha1="7lpBD3KMTWYvMqwn4W2yzkNVixj" width="211" height="74"></p>
<p>and in Firefox<br>
<img src="https://user-images.githubusercontent.com/1681525/181075572-4cf4dc57-1884-42e5-9123-fe7f9f1bb53c.png" alt="image" data-base62-sha1="kczzTmuMzN0SH6gm64fECtUrDkT" width="215" height="76"></p>

**Comment by omBratteng** [comment]
<p>After some investigation, it appears to be a Chrome issue and retina vs non-retina screens, not GitHub.</p>
<p>Non-retina:<br>
<img src="https://user-images.githubusercontent.com/1681525/181075575-2a266e01-f08d-45ca-9afe-796c18d8f2c9.png" alt="image" data-base62-sha1="tdia0AreGuLkYljT1G6C2PXInNI" width="585" height="47"></p>
<p>Retina:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/f/2f765c54ceae5c1d8c113fb7448855f2e59a8d04.png" data-download-href="/uploads/short-url/6LS1InJzLBK51ohq7i9qVwUYF6c.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/1681525/181075577-95ae4fb4-a28f-4b14-ac01-61dbdb84b9ab.png" alt="image" data-base62-sha1="6LS1InJzLBK51ohq7i9qVwUYF6c" width="690" height="41" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/f/2f765c54ceae5c1d8c113fb7448855f2e59a8d04_2_690x41.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/f/2f765c54ceae5c1d8c113fb7448855f2e59a8d04_2_1035x61.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/2/f/2f765c54ceae5c1d8c113fb7448855f2e59a8d04.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/f/2f765c54ceae5c1d8c113fb7448855f2e59a8d04_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1158×70 16.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
